spec/support/schemas/stores/update.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/Root",
+  "definitions": {
+    "Root": {
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
Another thing I like about https://jsonformatter.org/json-to-jsonschema is that it specifies both `"additionalProperties": false` and lists all properties as `required`. Most/all other JSON-to-JSON-Schema generators I tried don't specify both of those.